application-development/application-integration-and-process-automation/reusable-assets-cloud/README.md

```diff
@@ -0,0 +1,11 @@
+# Reusable Assets: Cloud
```

Comment:
@MarcelStraka This is a custom page under app integration. 

I suggested having here the product folders and their folders.

Reply:
These assets relate to several cloud services, this is why we have it under the root. We have the same in Confluence